lib/constants/source.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+const sources = {
+  user: 'User',
+  api: 'API',
+  edi: 'EDI',
+  marc: 'MARC',
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export const sourceOptions = Object.keys(sources).map(source => ({
+  value: sources[source],
+  label: <FormattedMessage id={`stripes-acq-components.sources.${source}`} />,
```

Comment:
FYI, this pattern is broken when value of source contains forbidden symbols, that can't be used as translation key.